spec/models/lan_spec.rb

```diff
@@ -1,19 +1,17 @@
 describe Lan do
-  before(:each) do
-    @lan      = FactoryGirl.create(:lan)
-    @vm       = FactoryGirl.create(:vm_vmware,       :hardware => FactoryGirl.create(:hardware, :guest_devices => [FactoryGirl.create(:guest_device_nic, :lan => @lan)]))
-    @template = FactoryGirl.create(:template_vmware, :hardware => FactoryGirl.create(:hardware, :guest_devices => [FactoryGirl.create(:guest_device_nic, :lan => @lan)]))
-  end
+  let(:lan) { FactoryGirl.create(:lan) }
+  let(:vm) { FactoryGirl.create(:vm_vmware,       :hardware => FactoryGirl.create(:hardware, :guest_devices => [FactoryGirl.create(:guest_device_nic, :lan => lan)])) }
```

Comment:
I think `vm` and `template` should be using `let!` to ensure that they are created early.  I think this was the intent of the before block.

Reply:
I have fixed bugs.